### DISCUSSION TITLE
Use extension on derived DataGrid type

### DISCUSSION THREAD

**Discussion by mike101200** [dotnet/DataGridExtensions#92] [Q&A] [answered]
In my project I use a custom DataGrid type (MyCellCommitDataGrid : DataGrid). My question is how can I use DataGridExtensions with it?

**Comment by tom-englert** [comment, marked as answer]
It's derived from `System.Windows.Controls.DataGrid`?

**Comment by mike101200** [comment]
Yes, but I just found out that is working OK, only VS was giving me an outline that something is not OK and was showing only partial methods on autocomplete, but at build it is working OK.
Sorry for bothering you.
